packages/rulesets/package.json

```diff
@@ -71,7 +71,7 @@
   },
   "dependencies": {
     "@apidevtools/json-schema-ref-parser": "^9.0.9",
-    "@stoplight/json-ref-resolver": "3.1.3",
+    "@stoplight/json-ref-resolver": "^3.1.6",
```

Comment:
- Failures when `3.1.3` and `3.1.6` were loaded side-by-side
- Unclear from git history why this dep was pinned when related deps floated